### ISSUE TITLE
Question: Animations Are Firing too Early in the Page

### ISSUE THREAD

**Issue by lisacowanyr** [johnpolacek/superscrollorama#144]
Hi John,

I have an animation posted at:

http://dev.digital-module.com/lisa/

It's kind of a mess right now, but if you look at the prop plane and the balloons, you'll get the gist of the problem I'm having. The prop plane works as I want it to. The balloons, however, fire too early. They shouldn't start rising until the user reaches that frame (around 4000 pixels), but all but the last 2 are already done when you get to that screen. triggerAtCenter is set to false.

I'm a noob at javascript, but I've done a lot of reading about GSAP. I think I kinda sorta understand how all of this works. Any guidance you can provide is greatly appreciated.

Thanks,
Lisa
